evalml/pipelines/utils.py

```diff
@@ -225,23 +226,25 @@ def generate_pipeline_code(element):
     return "\n".join(code_strings)
 
 
-def _make_stacked_ensemble_pipeline(input_pipelines, problem_type, random_state=0):
+def _make_stacked_ensemble_pipeline(input_pipelines, problem_type, n_jobs=-1, random_state=0):
     """
     Creates a pipeline with a stacked ensemble estimator.
 
     Arguments:
         input_pipelines (list(PipelineBase or subclass obj)): List of pipeline instances to use as the base estimators for the stacked ensemble.
             This must not be None or an empty list or else EnsembleMissingPipelinesError will be raised.
         problem_type (ProblemType): problem type of pipeline
+        n_jobs (int or None): Non-negative integer describing level of parallelism used for pipelines.
```

Comment:
nit-pick: I think using "non-negative" is confusing since -1 is allowed. I think saying "Integer describing...." is fine.